Tests/configure_and_test_integration_instances.py

```diff
@@ -668,18 +680,39 @@ def main():
         tests_for_iteration = []
     elif filter_configured and filtered_tests:
         tests_for_iteration = [test for test in tests if test.get('playbookID', '') in filtered_tests]
+
+    prints_manager.add_print_job("Before filter_tests_with_incompatible_version", print_warning, 0)
+    prints_manager.execute_thread_prints(0)
+
     tests_for_iteration = filter_tests_with_incompatible_version(tests_for_iteration, server_numeric_version,
                                                                  prints_manager)
     prints_manager.execute_thread_prints(0)
 
     # get a list of brand new integrations that way we filter them out to only configure instances
     # after updating content
-    new_integrations_names, modified_integrations_names = get_new_and_modified_integrations(git_sha1)
+    new_integrations_names, modified_integrations_names, name_to_path_dict = get_new_and_modified_integrations(git_sha1)
     if new_integrations_names:
         new_integrations_names_message = \
             'New Integrations Since Last Release:\n{}\n'.format('\n'.join(new_integrations_names))
         prints_manager.add_print_job(new_integrations_names_message, print_warning, 0)
     if modified_integrations_names:
+
+        # Test packs search and installation - beginning of infrastructure
+        threads_list = []
+        packs_in_progress = set()  # used to avoid double installation
+        threads_prints_manager = ParallelPrintsManager(len(modified_integrations_names))
+        for index, int_name in enumerate(modified_integrations_names):
+            client = demisto_client.configure(base_url=servers[0], username=username, password=password,
+                                              verify_ssl=False)
+            thread = Thread(target=search_and_install_pack_and_its_dependencies,
```

Comment:
did you test this? I am not sure it can work now because our dependencies are not populated.